src/pages/MenuPage.js

```diff
@@ -28,183 +27,208 @@ import {
+            <MenuButton text={navStrings.stocktake} onPress={toStocktakes} />
+          </InfoBadge>
         </View>
-        <View style={globalStyles.bottomContainer}>
+      </View>
+    ),
+    []
+  );
+
+  const ModulesSection = useCallback(
+    () => (
+      <View style={containerStyle}>
+        <ModulesImage style={image} />
+        <View>{usingDispensary && <MenuButton text="Dispensary" />}</View>
+      </View>
+    ),
+    []
+  );
+
+  const AdminRow = useCallback(
+    () => (
+      <View style={styles.bottomRow}>
+        <View style={styles.bottomIconView}>
           <Icon.Button
             name="power-off"
-            underlayColor="#888888"
-            iconStyle={localStyles.bottomIcon}
-            borderRadius={4}
+            iconStyle={bottomIcon}
             backgroundColor="rgba(255,255,255,0)"
             onPress={logOut}
           >
-            <Text style={localStyles.logOutText}>{navStrings.log_out}</Text>
+            <Text>{navStrings.log_out}</Text>
           </Icon.Button>
         </View>
+        {isInAdminMode && <MenuButton text="Realm Explorer" onPress={toRealmExplorer} />}
+        {isInAdminMode && <MenuButton text="Export Data" onPress={exportData} />}
       </View>
-    );
-  }
-}
+    ),
+    []
+  );
+
+  const ModuleLayout = useCallback(
+    () => (
+      <View style={styles.moduleTopRow}>
+        <View style={moduleRow}>
+          <CustomerSection />
+          <SupplierSection />
+        </View>
+        <View style={moduleRow}>
+          <StockSection />
+          <ModulesSection />
+        </View>
+      </View>
+    ),
+    []
+  );
+
+  const OriginalLayout = useCallback(
+    () => (
+      <View style={styles.originalTopRow}>
+        <CustomerSection />
+        <SupplierSection />
+        <StockSection />
+      </View>
+    ),
+    []
+  );
+
+  return (
+    <View style={{ ...appBackground }}>
+      {usingModules ? <ModuleLayout /> : <OriginalLayout />}
+      <AdminRow />
+    </View>
+  );
+};
+
+const styles = {
+  moduleTopRow: { flex: 9 },
+  originalTopRow: { flex: 9, flexDirection: 'row' },
+  moduleRow: { flex: 1, flexDirection: 'row' },
+  image: { height: 150, width: 150, marginBottom: 30 },
+  bottomIcon: { color: GREY },
+  bottomIconView: { flexGrow: 1, marginLeft: 100 },
+  bottomRow: {
+    flex: 1,
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'flex-end',
+  },
+  moduleContainer: {
+    alignItems: 'center',
+    borderColor: SHADOW_BORDER,
+    borderWidth: 1,
+    justifyContent: 'center',
+    flexDirection: 'row',
+    marginTop: 30,
+    paddingLeft: 50,
+    marginHorizontal: 15,
+    flex: 1,
+    backgroundColor: 'white',
+  },
+  originalContainer: {
+    backgroundColor: 'white',
+    alignItems: 'center',
+    borderColor: SHADOW_BORDER,
+    borderWidth: 1,
+    justifyContent: 'flex-start',
+    marginTop: 30,
+    paddingTop: 50,
+    marginHorizontal: 30,
+    flex: 1,
+  },
```

Comment:
There's a lot of similarity between the container styles. I was wondering why it isn't possible to have the same style instead of creating the two and that lead me to think about bringing together the common styles e.g.

```
  container: {
    alignItems: 'center',
    backgroundColor: 'white',
    borderColor: SHADOW_BORDER,
    borderWidth: 1,
    flex: 1,
    marginTop: 30,
    paddingTop: 50,
  },
  moduleContainer: {
    flexDirection: 'row',
    justifyContent: 'center',
    marginHorizontal: 15,
  },
  originalContainer: {
    justifyContent: 'flex-start',
    marginHorizontal: 30,
  },
```

and then something like this, which makes things a bit messy again - so may not be worth it.

```
  const { image, container, originalContainer, moduleContainer, bottomIcon, moduleRow } = styles;

  const containerStyle = Object.assign(
    {},
    container,
    usingModules ? moduleContainer : originalContainer
  );
```

Reply:
In general for styles I've found that creating multiple,  similar styles to just be easier to work with. Taken a bit far here, thanks 56b6dae 

Also what did you use to update the image? (thanks btw!)

Reply:
I used gimp for the image - there's something odd with the colour palette that I didn't figure out. worked around it by copying one of the other images and pasting in a new layer